### ISSUE TITLE
Works fine but sometimes the screen goes white 

### ISSUE THREAD

**Issue by jason-engage** [Telerik-Verified-Plugins/WKWebView#164]
Hi, since I installed version 0.6.0 I have a fairly random problem pop up from time to time. I'm using ionic/angular, with fairly long lists (ng-repeat), which can take a second or two to load up. If I go into the list item, then back to the list, sometimes the list page will go full white. I can still click on an item (although I cannot see it) and the item detail page will appear normally. And if I go back to the list it might appear fine. The problem seems to appear mostly if i'm going back and forth very fast (ie. testing the loading speeds). My device memory usage holds steady at about 25 mb and I'm still testing on ios 8.4 with iphone 6.

I'm not 100% sure its WkWebView, but if anyone else is having this issue please chime in. Thanks

**Comment by kaansoral** [comment]
Not entirely related, but these kind of issues exists when performance is in it's limits, it's something I experience, usually it's a rendering glitch that goes away

I've noticed these issues happening more frequently when I switched to the WKWebView, I was expecting the performance to skyrocket and these issues become a thing of the past, however, after the first trial, I returned back to UIWebView (iScroll+Masonry, +Maps, too much for the devices to handle)

It's obviously not a scientific experiment and not entirely related to your issue, but my .02

**Comment by ozexpert** [comment]
same for me.. this made me not use the wkwebview

**Comment by jason-engage** [comment]
@ozexpert Are you using angular? I'm noticing that it may have something to do with digest cycles...

**Comment by ozexpert** [comment]
yes. I'm using ionic framwork which is angular based.

**Comment by jason-engage** [comment]
Yes me too. I think its an issue with transitioning between views. I've been using $watch on  $rootScope.$$phase and only changing views when $$phase ==null (signifying that the digest cycle is over). I think that helps. It's important to wrap all your $state.go in $timeout() as well. I think I may have it conquered but I'm not 100% sure yet.

**Comment by ozexpert** [comment]
@jason-engage yap. I'll try that out later. There's some other issues with other plugins, and seems it's really an experimental stage, so will try out later.
